python/ray/data/_default_config.py

```diff
@@ -0,0 +1,2 @@
+# Default file shuffler class to use.
+DEFAULT_FILE_SHUFFLER = "ray.data.datasource.file_shuffler.SequentialFileShuffler"
```

Comment:
I prefer to store the class name here, because in the runtime, different objects will have different parameters. For example, Spark is doing this - https://github.com/apache/spark/blob/master/sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala#L2317-L2322 .